crates/bin/pcli/src/command/tx.rs

```diff
@@ -264,10 +264,15 @@ pub enum TxCmd {
         /// The selected fee tier to multiply the fee amount by.
         #[clap(short, long, default_value_t)]
         fee_tier: FeeTier,
-        /// Whether to use a Bech32(non-m) address for the withdrawal.
+        /// Whether to use a Bech32(non-m) address for the withdrawal (deprecated).
         /// Required for some chains for a successful acknowledgement.
         #[clap(long)]
         use_compat_address: bool,
```

Comment:
do we want to keep the `use_compat_address` for any reason or good to remove it from pcli?

Reply:
I think we could drop it from pcli

Reply:
Do we want to have a `pcli view address --transparent-address` similar to `pcli view address --compat`?

This will be handy for testing roundtrips.